server/test/unit/server/db/test_model.py

```diff
@@ -1,19 +1,232 @@
 """
 Tests for the pulp.server.db.model module.
 """
+from tempfile import mkdtemp
+import shutil
+import os
+
+from mock import patch
+
 try:
     import unittest2 as unittest
 except ImportError:
     import unittest
 
-
 from mongoengine import DateTimeField, DictField, Document, IntField, StringField
 
+from pulp.common import error_codes
+from pulp.devel.unit.util import touch
+from pulp.server.exceptions import PulpCodedException
 from pulp.server.db import model
 from pulp.server.db.model.base import CriteriaQuerySet
 from pulp.server.db.model.fields import ISO8601StringField
 
 
+class TestContentUnit(unittest.TestCase):
+    """
+    Test ContentUnit model
+    """
+
+    def setUp(self):
+        self.working_dir = mkdtemp()
```

Comment:
I expected this setUp and tearDown to use a lot of mocking and never actually touch the filesystem.